docs/guide/yaml/test/test-entities.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Blueprint Test Entities
+title_in_menu: Test Entities
+layout: website-normal
+---
+
+{% include fields.md %}
+
+## Structural Test Entities
+
+### TestCase
+The `TestCase` entity acts as a container for a list child entities which are started *sequentially*.
+{% highlight yaml %}
+{% readj example_yaml/testcase-entity.yaml %}
+{% endhighlight %}
+The ability to sequentially run entities is used to allow applications to fully deploy before attempting to start any test entities.
+
+If your application blueprint consists of multiple services then you will also require the `ParallelTestCase` entity.
```

Comment:
Yes, that's what I was thinking of. That feels more like you are adding the test assertions, rather than having to "restructure" their app by putting it inside a `ParallelTest entity`.